src/sonic-yang-models/yang-models/sonic-kdump.yang

```diff
@@ -54,6 +54,23 @@ module sonic-kdump {
                     description
                         "Maximum number of Kernel Core files Stored";
                 }
+                leaf remote {
+                    type boolean;
+                    description
+                        "Enable or Disable the Kdump remote ssh mechanism";
+                }
+
+                leaf ssh_string {
+                    description  "Remote ssh connection string";
+                    type string;
+                }
+
+                leaf ssh_path {
+                    description  "Remote ssh private key path";
+                    type string;
+                }
```

Comment:
Could you update these new changes to doc/Configuration.md?

Reply:
updated, please review